specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-09-15/cosmos-db.json

```diff
@@ -6166,6 +6166,10 @@
         "disableLocalAuth": {
           "description": "Opt-out of local authentication and ensure only MSI and AAD can be used exclusively for authentication.",
           "type": "boolean"
+        },
+        "capacity": {
+          "description": "The object that represents all properties related to capacity enforcement on an account.",
```

Comment:
recently, linter started complaining if there is `type` is not defined. (even if its defined inside the property definition). So if we get any errors, we might have to add. Just a heads up. :)